### PR TITLE
Added functionality to export query results with or without table hea…

### PR DESCRIPTION
**Pull request by jnazaren**
…ders as CSV

### CODE REVIEW

**Comment by jnazaren** [comment]
@sunnygoyal I really enjoy using this app! I found that this would be a useful feature to add, as a lot of other DBMS apps have it - this update gives the user the possibility to not only execute an SQL query, but also export the query as a CSV file (optionally with the result header). Let me know if you have any comments or questions!
- Also, would it be possible to implement this in the online demo version?

File: index.html [hunk 1]
```diff
@@ -101,6 +101,11 @@
         <div class="card" id="cardSqlInput">
           <textarea id="contentCodeMirror" name="contentCodeMirror" placeholder="Enter SQL here..." i18n-id="placeholder_sql" i18n-attr="placeholder"></textarea>
           <a class="waves-effect waves-light orange darken-1 btn" id="btnExecuteSql" i18n-id="btn_run_sql">Run SQL</a>
+		  <a class="waves-effect waves-light orange darken-1 btn" id="btnSaveSql" i18n-id="btn_save_sql">Save Result as CSV</a>
```
Comment (sunnygoyal) [inline index.html]: fix indentation

File: worker/sql.worker.js [hunk 2]
```diff
@@ -87,6 +87,9 @@ var handleCommand = function(data) {
     case "export" : {
       return exportData(data);
     }
+	case "exportResult" : {
```
Comment (sunnygoyal) [inline worker/sql.worker.js]: indentaiton

File: worker/sql.worker.js [hunk 3]
```diff
@@ -207,6 +210,40 @@ var exportData = function(args) {
   return {blob: data};
 }
 
+var exportResult = function(args) {
+
```
Comment (sunnygoyal) [inline worker/sql.worker.js:214]: indentation: 2 spaces instead of tab?

**Comment by sunnygoyal** [comment]
The demo app will get updated automatically as it uses the same source

**Comment by jnazaren** [comment]
@sunnygoyal Ok, I've fixed the indentation - sorry about that.

**Comment by sunnygoyal** [comment]
Just realized, the demo does not update automatically. Will update it and the chrome app later today (or by weekend)

**Comment by jnazaren** [comment]
Sounds good, Thank you!!

@sunnygoyal Let me know when you've updated it

**Pull request merged**